### ISSUE TITLE
Add way to view more images

### ISSUE THREAD

**Issue by cassidyjames** [cassidyjames/jppartyrentals.com#34]
We have several images for each bounce house/inflatable, but only one small-ish photo each. We should find a way to show multiple photos for each one if they exist.

**Comment by jprentals** [comment]
Another option is i have been thinking about doing "walkaround" videos of each item and adding to our YouTube channel and then adding the youtube link to that video under the item.

Jordan Henry
JP Party rentals
www.jppartyrentals.com
515-988-7604

**Comment by cassidyjames** [comment]
Yeah that could also be a good option! Let me know if you want to go 
that route and we can add those links.

**Comment by jprentals** [comment]
Will do

Jordan Henry
JP Party rentals
www.jppartyrentals.com
515-988-7604
